### PR TITLE
google-sheets: fix tests to work with updated component

### PR DESCRIPTION
**Pull request by mmelko**
//test: `@google-sheets`
expect to fail 2 tests .. one is disabled

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
@mmelko probably a rebase is needed due to the installation updates

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google Sheets Connector | Append messages from DB 
Google Sheets Connector | Copy Big SS to DB and back using split/aggregate 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmelko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google Sheets Connector | Append messages from DB 
Google Sheets Connector | Copy Big SS to DB and back using split/aggregate 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google Sheets Connector | Copy Big SS to DB and back using split/aggregate 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google Sheets Connector | create pivottable from sample data 
Google Sheets Connector | Copy big spreadsheet 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/google-sheets.feature [hunk 1]
```diff
@@ -289,14 +295,15 @@ Feature: Google Sheets Connector
...
-    And sleep for 60 seconds
+    And sleep for 10 seconds
     Then click on the "Done" button
 
     When publish integration
     And set integration name "pivot-table"
     And publish integration
     And navigate to the "Integrations" page
     And wait until integration "pivot-table" gets into "Running" state
+    And sleep for 10 seconds
```
[the remark this answers is not in the document]
Comment (mmelko) [reply ui-tests/src/test/resources/features/integrations/google-sheets.feature]: just to get some time for create pivot table in the spreadsheet

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google Sheets Connector | Copy big spreadsheet 
```

**Comment by mmelko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**